components/TARGET_PSA/TARGET_TFM/COMPONENT_SPE/secure_fw/spm/spm_api.h

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2017-2019, Arm Limited. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+
+#ifndef __SPM_API_H__
+#define __SPM_API_H__
+
+/* This file contains the apis exported by the SPM to tfm core */
+#include "spm_partition_defs.h"
+#include "secure_fw/core/tfm_secure_api.h"
+
+#define SPM_INVALID_PARTITION_IDX     (~0U)
+
```

Comment:
Does these enums require Doxygen comments?

Reply:
This file is also imported.